Gemfile

```diff
@@ -45,6 +45,7 @@ source 'https://rubygems.org' do
     gem 'phantomjs-binaries'
     gem 'database_cleaner'
     gem 'chronic'
+    gem 'scss-lint', '~> 0.30'
```

Comment:
stuck with the same version of the linter as pension_guidance, as I was having some issues running the rake task with the latest version of the linter installed